cibuildwheel/macos.py

```diff
@@ -137,13 +139,22 @@ def install_cpython(tmp: Path, version: str, url: str) -> Path:
             # download the pkg
             download(url, pkg_path)
             # install
-            call("sudo", "installer", "-pkg", pkg_path, "-target", "/")
+            args = []
+            if version.startswith("3.13"):
+                # Python 3.13 is the first version to have a free-threading option
+                args += ["-applyChoiceChangesXML", str(free_thread_enable_313.resolve())]
+            call("sudo", "installer", "-pkg", pkg_path, *args, "-target", "/")
             pkg_path.unlink()
             env = os.environ.copy()
             env["PIP_DISABLE_PIP_VERSION_CHECK"] = "1"
-            call(installation_path / "bin" / "python3", install_certifi_script, env=env)
 
-    return installation_path / "bin" / "python3"
+            if free_threading:
+                call(installation_path / f"bin/python{version}t", "-m", "ensurepip", env=env)
+                call(installation_path / f"bin/python{version}t", install_certifi_script, env=env)
+            else:
+                call(installation_path / "bin/python3", install_certifi_script, env=env)
```

Comment:
```suggestion
            if free_threading:
                call(installation_path / f"bin/python{version}t", "-m", "ensurepip", env=env)
            call(installation_path / "bin/python3", install_certifi_script, env=env)
```
I think we don't need to run  `install_certifi_script` with `python3.13t`. openssl is shared between `python3.13` & `python3.13t`.
```
cibuildwheel % python3.13t -c 'import ssl; print(ssl.get_default_verify_paths())'
DefaultVerifyPaths(cafile='/Library/Frameworks/Python.framework/Versions/3.13/etc/openssl/cert.pem', capath=None, openssl_cafile_env='SSL_CERT_FILE', openssl_cafile='/Library/Frameworks/Python.framework/Versions/3.13/etc/openssl/cert.pem', openssl_capath_env='SSL_CERT_DIR', openssl_capath='/Library/Frameworks/Python.framework/Versions/3.13/etc/openssl/certs')
```

Reply:
Oops, no such file `'/Library/Frameworks/PythonT.framework/Versions/3.13/bin/python3'`

Reply:
I've reverted this, I think it's fine to use python3.13t even if they are shared, better than keeping both installation paths.